apps/judicial-system/backend/src/app/formatters/formatters.ts

```diff
@@ -189,6 +190,32 @@ export function formatCourtDateEmailNotification(
   )}. Dómsalur: ${courtRoom}.`
 }
 
-export function formatCourtDateNotificationCondition(courtDate: Date) {
+export function formatPrisonCourtDateEmailNotification(
+  court: string,
+  courtDate: Date,
+  accusedGender: CaseGender,
+  requestedCustodyEndDate: Date,
+  isolation: boolean,
+): string {
+  const courtText = court.replace('dómur', 'dóms')
+  const courtDateText = formatDate(courtDate, 'PPPp')
+  const accusedGenderText =
+    accusedGender === CaseGender.OTHER
+      ? 'K'
+      : `Sakborningur er ${
+          accusedGender === CaseGender.MALE ? 'karl' : 'kona'
+        } og k`
```

Comment:
Why doesn't this end with a space and  " og k" placed in line 216?

Reply:
Rest of the sentence is there :)

"...rafist er gæsluvarðhalds til..."

This is covered in tests, which pass.

Reply:
I saw the rest of the sentence in line 216, just thought it was weird 🤷‍♂️  I'd argue that you should change it for better readability...

Reply:
You have a point there :) Will take a look.